TESTS/dev_mgmt/connect/main.cpp

```diff
@@ -135,51 +133,20 @@ void spdmc_testsuite_connect(void) {
 
     test_case_finish("Connect to " TEST_NETWORK_TYPE, iteration + (net_status == 0), (net_status != 0));
 
-    test_case_start("Initialize " TEST_BLOCK_DEVICE_TYPE "+" TEST_FILESYSTEM_TYPE, 2);
+    test_case_start("Initialize Storage", 2);
```

Comment:
We should rather report the block device type, i.e. there could be multiple block storage option and the user should be aware which one is being used. Could you restore the behavior?